### PR TITLE
Issue #1163 - Convert ResistorColors to use template

### PR DESCRIPTION
**Pull request by Mailkov**
@moollaza I don't use fix becuse this PR does not closes Issue 1163

-----
IA Page: https://duck.co/ia/view/resistor_colors

### CODE REVIEW

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=4.7k ohm)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/resistor_colors_mobile.png)

File: share/goodie/resistor_colors/content.handlebars [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{{#each resistor_bands}}
+    <span class='resistor-band {{class}}'>{{name}} {{html_prefix}}{{display_digit}}</span>
+{{/each}}
+<br>
+<a href='http://resisto.rs/#{{formatted_value}}' class='zci__more-at'>More at resisto.rs</a>
```
Comment (moollaza) [inline share/goodie/resistor_colors/content.handlebars]: Our templates will actually create this for you :smiley: 

You need to create a `meta` property in the Perl, and enable the `moreAt` flag:

```perl
    structured_answer => {
        id => 'resistor_colors',
        name => 'Answer',
        meta => {
            sourceName => "resisto.rs",
            sourceUrl => "http://resisto.rs/$formatted_value"
        },
        data => {
            title => $title,
            subtitle => $bands . ' Bands',
            resistor_bands => \@resistor_bands,
            formatted_value => $formatted_value
        },
        templates => {
            group => 'text',
            item => 0,
            options => {
                content => 'DDH.resistor_colors.content',
                moreAt => 1
            }
        }
    };
};
```

File: share/goodie/resistor_colors/resistor_colors.css [hunk 2]
```diff
@@ -6,56 +6,60 @@
     color: #333;
 }
 
-.zci--answer .zci--resistor-colors .resistor-band.silver {
+.zci--resistor_colors .resistor-band.silver {
```
Comment (moollaza) [inline share/goodie/resistor_colors/resistor_colors.css]: We now have our own CSS classes for all these colours. We can use those names in the Perl and ditch all this CSS.

https://duckduckgo.com/styleguide

In our perl we'll want to use, for example, `bg-clr--gold` to set the background colors and `tx-clr--white`, `tx-clr--slate` for white/black text.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=4.7k ohm)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/resistor_colors_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=4.7k ohm)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/resistor_colors_mobile.png)

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

**Pull request merged**